crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -273,6 +281,13 @@ pub struct PipelineCache {
 }
 
 impl PipelineCache {
+    pub fn count_ready(&self) -> usize {
```

Comment:
This would be used by users to check if / how many are ready? (Just to make sure I fully understand)

Reply:
Maybe it would be better to just expose a (read-only) iterator over the pipelines so consumers can interact with / query them as they like?

Reply:
> This would be used by users to check if / how many are ready? (Just to make sure I fully understand)

Yup!

As Cart suggested, I changed it to an iterator, that will be a lot more flexible